### PR TITLE
Added ETS form validation in teamengine.

### LINKED ISSUE #354: Add ETS form validation
Reference from: https://github.com/opengeospatial/ets-kml22/issues/11#issuecomment-416521111

### PR DESCRIPTION
**Pull request by keshavnangare**
Fixed #354.

### CODE REVIEW

File: teamengine-web/src/main/webapp/test.jsp [hunk 1]
```diff
@@ -167,8 +168,34 @@ if (mode.equals("retest") || mode.equals("resume") || mode.equals("cache")) {
...
+        				return false;
+        			}
+        			if (iutdoc != null && iutdoc != "" && iuturi != null && iuturi != "") {
+        				alert("Provide only one input either capability URL or document!");
+        				return false;
+        			}
+        		}
+                
+                // Reset iut_uri if the file is uploaded.
+                function resetIutValue(iut_uri, iut_doc){
```
Comment (lgoltz) [inline teamengine-web/src/main/webapp/test.jsp]: Please remove this function.

**Comment by keshav-nangare** [comment]
Updated the PR with suggested changes.

**Review by bpross-52n — APPROVED** [review]
Tested with ets-wfs20. I guess we need some documentation about how to use this.

**Comment by dstenger** [comment]
New issue for documentation was created: https://github.com/opengeospatial/teamengine/issues/503

**Pull request merged**